generators/entity-client/templates/angular/src/test/javascript/e2e/entities/entity.spec.ts.ejs

```diff
@@ -107,7 +107,7 @@ describe('<%= entityClass %> e2e test', () => {
         <%= entityInstance %>DialogPage.set<%= fieldNameCapitalized %>Input('2000-12-31');
         expect(<%= entityInstance %>DialogPage.get<%= fieldNameCapitalized %>Input()).toMatch('2000-12-31');
         <%_ } else if (['Instant', 'ZonedDateTime'].includes(fieldType)) { _%>
-        <%= entityInstance %>DialogPage.set<%= fieldNameCapitalized %>Input(12310020012301);
+        <%= entityInstance %>DialogPage.set<%= fieldNameCapitalized %>Input('31/12/2001' + protractor.Key.TAB + '02:30');
```

Comment:
nice, I was trying to figure this out earlier and gave up

Reply:
I think we have a problem then, chrome 64 on OSx does not display AM-PM
<img width="804" alt="capture d ecran 2018-02-26 a 21 00 15" src="https://user-images.githubusercontent.com/792925/36692348-19ed01b0-1b38-11e8-87aa-bad0373bcf21.png">.

I've got to make protractor tests a smarter